library/src/main/java/com/alamkanak/weekview/WeekViewUtil.java

```diff
@@ -67,4 +69,10 @@ public static int daysBetween(Calendar dateOne, Calendar dateTwo){
                 (dateOne.getTimeInMillis() + dateOne.getTimeZone().getOffset(dateOne.getTimeInMillis())))
                         / (1000 * 60 * 60 * 24f));
     }
+
+    public static int getTextColor(int color) {
```

Comment:
I think this function shouldn't be included in the weekview. I think it should be possible to implement this method with an interface, so everyone can use it's own implementation.